src/StructuredLogger/StructuredLogger.csproj

```diff
@@ -28,6 +28,8 @@
     <PackageReference Include="Microsoft.Build.Framework" Version="$(MSBuildPackageVersion)" />
     <PackageReference Include="Microsoft.Build.Utilities.Core" Version="$(MSBuildPackageVersion)" />
     <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.0.0" PrivateAssets="all" />
+    <PackageReference Include="MSBuild.BinlogRedactor.SensitiveDataDetector" Version="0.0.4-beta" />
+    <PackageReference Include="DotUtils.StreamUtils" Version="0.0.6" />

```

Comment:
I'm hesitant about adding dependencies to the core StructuredLogger.dll and the MSBuild.StructuredLogger NuGet package. I think StreamUtils is only used for a few helpers, such as `ReadToEnd`, `SkipBytes`, `Concat` and `Slice`. It is of course neat to be able to have primitives like these to operate on streams, I'm just wondering if it'd be possible to inline this logic into a .cs file and include that into the project to avoid a dependency. 

Otherwise we'd have to find a way to ship a non-prelease NuGet package depending on a couple of pre-release ones.